accounts/admin.py

```
@@ -1,3 +1,18 @@
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    model = User
+    list_display = ('username', 'email', 'role', 'is_staff')
+    fieldsets = UserAdmin.fieldsets + (
+        ('Additional Details', {
+            'fields': [
+                'role',
+                'uuid'
+            ],
+        }),
+    )
```

Ruff (0.8.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)